src/verify/process.rs

```diff
@@ -214,6 +214,7 @@ pub fn process_verify(args: VerifyArgs) -> Result<()> {
     let cluster = match get_cluster(program.rpc())? {
         Cluster::Devnet => "devnet",
         Cluster::Mainnet => "mainnet",
+        Cluster::Localnet => "",
```

Comment:
should this say localhost instead of empty quotes

Reply:
In this case, no, because this is preparing the cluster for inserting into the SolanEyes link below. SolanEyes doesn't support localhost AFAIK so we want to leave this empty.